tests/tck-build-logic/src/main/resources/contributing/questions.json

```diff
@@ -0,0 +1,37 @@
+[
+  {
+    "question-key": "coordinates",
+    "question": "What library you want to support? Maven coordinates: ",
+    "help": "Maven coordinates consist of three parts in the following format \"groupId:artifactId:version\". For more information visit: https://maven.apache.org/repositories/artifacts.html"
+  },
+  {
+    "question-key": "testsLocation",
+    "question": "Where are your tests implemented? Absolute path to the directory containing java packages where you implemented tests: ",
```

Comment:
I'd reword that. It wasn't clear to me that those are the existing tests for the library I want to add.